Sources/FlowLayoutStack.swift

```diff
@@ -0,0 +1,122 @@
+//  Copyright © 2024 SEEK. All rights reserved.
+//
+
+open class FlowLayoutStack: Stack {
+    public var thingsToStack: [Stackable]
+    public var spacing: CGFloat
+    public var lineSpacing: CGFloat
+    // layoutMargins have not yet been implemented
+    public var layoutMargins: UIEdgeInsets = .zero
+    public var width: CGFloat?
+    private var itemSpacing: CGFloat {
+        spacing
+    }
+    
+    public init(
+        itemSpacing: CGFloat = 0.0,
+        lineSpacing: CGFloat = 0.0,
+        width: CGFloat? = nil,
+        thingsToStack: [Stackable]
+    ) {
+        self.spacing = itemSpacing
+        self.lineSpacing = lineSpacing
+        self.layoutMargins = .zero
+        self.width = width
+        self.thingsToStack = thingsToStack
+    }
+    
+    public convenience init(
+        itemSpacing: CGFloat = 0.0,
+        lineSpacing: CGFloat = 0.0,
+        width: CGFloat? = nil,
+        thingsToStack: () -> [Stackable]
+    ) {
+        self.init(
+            itemSpacing: itemSpacing,
+            lineSpacing: lineSpacing,
+            width: width,
+            thingsToStack: thingsToStack()
+        )
+    }
+    
+    open func framesForLayout(_ width: CGFloat, origin: CGPoint) -> [CGRect] {
+        var frames: [CGRect] = []
+        var currentY = origin.y
+        var currentX = origin.x
+
+        func moveToNextRow() {
```

Comment:
Having an inline func made the code around line 64 a little bit easier to digest without having to add a comment.  If we think it is obvious enough more than happy to remove and inline this into lines 64 etc.